src/codeExamples/replacingFlexContainer.stories.mdx

```diff
@@ -0,0 +1,142 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="code examples/replacing FlexContainer" />
+
+# Replacing FlexContainer
+
+One of the most universal tools in the design system toolbox was FlexContainer. It could turn into a row or column, and it was using the parameters the same way as CSS. It was easy to use and easy to remember.
+But it also required more definitions for each element, so we have decided to use more predefined layout primitives - Inline, Stack, Center, and Cluster. And we added some extra features.
+
+First a short review of the replacement components called layout primitives.
+
+## [Stack](?path=/docs/layout-primitives-stack--playground)
+
+Stack is a flex container that has predefined flex-direction to a column. Apart from the standard flexbox column, it allows you to add the same size gaps between the elements or insert wider gap (splitAt) after a defined element. 
+Property isRecursive allows you to insert another element with multiple children and apply the same rules to it
+
+## [Inline](?path=/docs/layout-primitives-inline--playground)
+
+Inline is a flex container that has predefined flex-direction to a row. You can adjust justify (justifyContent), align (alignItems), gap size, and stretch.
+
+## [Center](?path=/docs/layout-primitives-center--playground)
+
+Center does what you think it does - it aligns its children to the center horizontally and vertically.
+
+## [Cluster](?path=/docs/layout-primitives-cluster--playground)
+
+Cluster component is quite like inline component, it just allows for wrapping if the width is too small. This component is a good option for navigation.
+
+## Examples
+
+Let's take a look at replacement examples
+
+1.
+
+```jsx
+  <FlexContainer justifyContent="flex-start" alignItems="center">
+    <IhsIcon src={ihsLogo} alt="IHS Markit Logo" />
+    <IhsText src={ihsText} alt="IHS Markit" />
+  </FlexContainer>
+```
+
+FlexContainer is by default row, so it should turn to Inline instead.
+Replace it with inline:
+
+```jsx
+  <Inline justify="flex-start" align="center">
```

Comment:
I think `Inline` is by default `flex-start` so the `justify` prop can be omitted